your_daily_dose_malware/main.py

```diff
@@ -1,44 +1,49 @@
 import os
-
-import click
-from commands.scraper import scraper
+import argparse
+from rich.console import Console
+from your_daily_dose_malware.commands.utils.retrieve_malware import (all_most_recent,
+                                                                     hundred_most_recent)
 from dotenv import load_dotenv
-from commands.utils.retrieve_malware import (
-                            all_most_recent, hundred_most_recent)
 
 
 load_dotenv()
 
 
-@click.command(
-    help="""
-    Download either hundred recent malware's uploaded within the last 60 min or
-    all the recent malware's uploaded within the last 48 hours sha256 hash
-    by using api from 'https://mb-api.abuse.ch/api/v1/'
-
-    -s256 or --by-sha256: get the most recent sha256 hashes from
-    'https://bazaar.abuse.ch/export/txt/sha256/recent' save them in
-    sha256_names.txt then for each file  download all malware's in zip file
-
-    -hr or --hundred-recent: get the most recent hundred recent malware's
-    uploaded within the last 60 min
-    """
-)
-@click.option("-s256", "--by-sha256", is_flag=True)
-@click.option("-hr", "--hundred-recent", is_flag=True)
-def run_scrapper(by_sha256, hundred_recent):
-    headers = {"API-KEY": os.getenv("API_KEY")}
+def scrap_malware_bazaar(hundred_recent):
+    headers = {"API-KEY": os.getenv("MwBz_API_KEY")}
     if hundred_recent:
         hundred_most_recent(headers)
-    elif by_sha256:
-        all_most_recent(headers)
     else:
-        click.echo(
-            "	No selector provided. Please use either by_sha256, "
-            "hundred_recent as selector",
-        )
+        all_most_recent(headers)
+
+
+def run_scrapper(args):
+    scrap_malware_bazaar(args.hundred_recent)
+
+
+def main():
+
+    parser = argparse.ArgumentParser(
+        description="Download recent samples from multiple OSINT provider backends"
+                    "MalwareBazaar: all the recent recent malware's uploaded within the last 48 hours sha256 hash"
+                    "MalwareBazaar: By using api from 'https://mb-api.abuse.ch/api/v1/'"
+    )
+    parser.add_argument('-MwBz_48H', '--last-48H', action='store_true',
```

Comment:
Modified parameters, we will adding more backends, so tried to use a convention of names we can continue in the future.

Reply:
i find your flags too long maybe you could keep the same case eitheir min or maj. It's could be a good idea too have both short and  explicit flags

Reply:
Lets rewrite this flags when we add more backends to the tool, and we can clearly see all the modifiers that would be useful for the user.